modules/nf-core/shigapass/main.nf

```diff
@@ -0,0 +1,60 @@
+process SHIGAPASS {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/shigapass%3A1.5.0--hdfd78af_0':
+        'biocontainers/shigapass:1.5.0--hdfd78af_0' }"
+
+    input:
+    tuple val(meta), path(assembly)
+
+    output:
+    tuple val(meta), path("*.csv"), emit: report
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    # Convert our genome path to a file with a path in it
+    readlink -f $assembly > ${assembly}_tmp.txt
```

Comment:
Why do you need this? Does it work on cloud?

Reply:
ShigaPass takes a file with a path to one or more genomes as an input - giving an individual path as an input is not an option. Therefore this line creates a dummy file so the input can be handed to the module as a path to a fasta. I figured this would probably also enable paralellised shigapass runs which is a bonus for pipeline efficiency.

What do you think of this approach and do you have a suggestion for an alternative?

Probably I could just do echo instead of readlink -f; I figured readlink might be useful for dynamic pathing.

Reply:
I think you probably just want to use `ls`, because all the files should already be in this directory.

Reply:
Done!